tests/common.go

```diff
@@ -11340,3 +11340,45 @@ func PrereqForNodeDecomm(nodeToDecommission node.Node, suspendedScheds []*storka
 	}
 	return nil
 }
+
+// GetNodeFromIPAddress returns node details from the provided IP Address
+func GetNodeFromIPAddress(ipaddress string) (*node.Node, error) {
+	for _, eachNode := range node.GetNodes() {
+		log.Infof(fmt.Sprintf("Comparing [%v] with [%v]", eachNode.GetMgmtIp(), ipaddress))
+		if eachNode.GetMgmtIp() == ipaddress {
+			log.Infof("Matched IP Address [%v]", eachNode.MgmtIp)
+			return &eachNode, nil
+		}
+	}
+	return nil, fmt.Errorf("Unable to fetch Ipaddress for not ipaddress [%v]", ipaddress)
```

Comment:
shouldn't the error message be `fmt.Errorf("Unable to fetch Node details for ipaddress [%v]", ipaddress)`?